app/retail/templates/emails/bounty_hypercharged.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2020 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize %}
+
+{% block content %}
+
+<h1>{% trans "Checkout this bounty worth" %} {{ bounty.value_true|floatformat }} {{ bounty.token_name }}</h1>
+<br>
+
+<p>Work on "{{ bounty.title }}" and receive {{ bounty.value_true|floatformat }} {{ bounty.token_name }}</p>
+
+<p>{{ bounty.issue_description_text|truncatechars:500 }}</p>
+
+<div style="margin-bottom: 1em; margin-top: 1em;">
+  <a class="button" href="{{ bounty.absolute_url }}">{% trans "View on Gitcoin.co" %}</a>
+</div>
+
+<p>
+{% trans "You are receiving this email because your email address is on the notification list for this funded issue." %}  <a href="{% url 'email_settings' subscriber.priv %}">{% trans "Manage Email Settings" %}</a>.
```

Comment:
Is this language regarding the notification list what we actually want to use? @PixelantDesign @frankchen07

Reply:
i can sub in the language we want to use